.github/workflows/ci.yml

```diff
@@ -64,52 +89,92 @@ jobs:
           chmod +x ./cc-test-reporter
           ./cc-test-reporter before-build
       - name: Setup Database
-        run: |
-          bundle exec rake parallel:load_schema
+        run: bundle exec rake parallel:load_schema
       - name: Check for untracked changes in schema.rb
         uses: rootstrap/check_untracked_changes@v1
         with:
           path: "./db/schema.rb"
       - name: Get CPU info
         id: cpu_info
-        run: |
-          echo "cpu_cores=$(nproc)" >> $GITHUB_ENV
+        run: echo "cpu_cores=$(nproc)" >> $GITHUB_ENV
       - name: Run Tests
         env:
           KNAPSACK_CI_NODE_TOTAL: ${{ matrix.ci_node_total }}
           KNAPSACK_CI_NODE_INDEX: ${{ matrix.ci_node_index }}
           PARALLEL_TESTS_CONCURRENCY: ${{ env.cpu_cores }}
-        run: |
-          bundle exec parallel_rspec -n $PARALLEL_TESTS_CONCURRENCY -e './bin/parallel_tests'
+          OPENAPI: ${{ env.OPENAPI }}
```

Comment:
need to remove this, otherwise the ENV always gets set (even if empty) and the docs are regenerated.

By removing this, the ENV will only be set when it corresponds in line 80